src/Aspire.Hosting/ApplicationModel/ResourceExtensions.cs

```diff
@@ -576,29 +576,29 @@ public static int GetReplicaCount(this IResource resource)
     /// there are multiple compute environments and a compute environment is not explicitly specified.
     /// </summary>
 #pragma warning disable ASPIRECOMPUTE001 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
-    public static DeploymentTargetAnnotation? GetDeploymentTargetAnnotation(this IResource resource, IComputeEnvironmentResource? computeEnvironmentResource = null)
+    public static DeploymentTargetAnnotation? GetDeploymentTargetAnnotation(this IResource resource, IComputeEnvironmentResource? targetComputeEnvironment = null)
```

Comment:
Can we add some xml docs for the `targetComputeEnvironment` parameter?

Reply:
Hmmm, I'm surprised this didn't fail.